### DISCUSSION TITLE
Git Trailers

### DISCUSSION THREAD

**Discussion by cdenneen** [renovatebot/renovate#28047] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

I'm looking for how to add `--trailer` to the renovate commits.

**Comment by rarkins** [comment]
I think you can add anything you need using the `commitBody` option: https://docs.renovatebot.com/configuration-options/#commitbody

**Comment by felipecrs** [comment]
I proposed a PR for this:

- https://github.com/renovatebot/renovate/pull/44651

I explained in the description why I think `commitBody` is not good enough.
